src/store/store.ts

```diff
@@ -0,0 +1,43 @@
+import {configureStore} from "@reduxjs/toolkit";
+import {TypedUseSelectorHook, useDispatch, useSelector} from "react-redux";
+import {
+  authReducer,
+  boardReducer,
+  boardReactionsReducer,
+  columnsReducer,
+  notesReducer,
+  participantsReducer,
+  reactionsReducer,
+  requestsReducer,
+  skinToneReducer,
+  viewReducer,
+  votesReducer,
+  votingsReducer,
+} from "./features";
+
+const rootReducer = {
+  auth: authReducer,
+  board: boardReducer,
+  boardReactions: boardReactionsReducer,
+  columns: columnsReducer,
+  notes: notesReducer,
+  participants: participantsReducer,
+  reactions: reactionsReducer,
+  requests: requestsReducer,
+  skinTone: skinToneReducer,
+  view: viewReducer,
+  votes: votesReducer,
+  votings: votingsReducer,
+};
+
+export const store = configureStore({
+  reducer: rootReducer,
+  middleware: (getDefaultMiddleware) => getDefaultMiddleware(),
```

Comment:
Can we remove this line as we are not passing any custom middlewares ?

Reply:
Yes, you're right. I was thinking of implementing some kind of retry mechanism when a thunk fails using middleware, but didn't end up doing it. So for now, this line can be removed 👍